### ISSUE TITLE
Update/improve temporal filtering 

### ISSUE THREAD

**Issue by tsalo** [physiopy/peakdet#27]
Currently, I believe that there are hardcoded cutoffs for different modalities, but we probably want to be able to calculate cutoffs from sequence parameters as well. @62442katieb, I think this is your area of expertise.

**Comment by rmarkello** [comment]
I don't believe those modalities actually do anything right—I think they're vestiges from a former iteration of the codebase. I kept them as a reference for myself for what cutoffs were reasonable (though, as you accurately note, that will depend on sampling frequency!). 

They could definitely be improved to actually do stuff, though!! 💯

**Comment by 62442katieb** [comment]
I haven't familiarized myself with the codebase yet, but once I do, I'm definitely interested in looking through and adding/tweaking some filtering based on what I've been learning the past couple months!

+1 for actually doing stuff! 😄

**Comment by rmarkello** [comment]
The [ReadTheDocs site](https://peakdet.readthedocs.io/en/latest/) is still live! Though I don't think any _new_ changes will get pushed to that since it was hosted from my user account and not the `physiopy` org (I don't know how RTD handles ownership transfers 🤷‍♂️).

Anyway, happy to help people orient themselves to `peakdet`! I won't be contributing anything new anytime soon (see: for the remainder of grad school 😅) but am happy to answer questions.
